actions/mixed-mode-test/action.yml

```diff
@@ -0,0 +1,38 @@
+name: Run mixed mode tests for release
+
+inputs:
+  update_type:
+    description: 'One of MAJOR, MINOR, BUILD, PATCH'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Base Environment
+      uses: ./actions/setup-base-env
+    - name: Setup FDB
+      uses: ./actions/setup-fdb
+    - name: Run Gradle Test
+      uses: ./actions/gradle-test
+      with:
+        gradle_command: mixedModeTest
+        gradle_args: -PreleaseBuild=false -PpublishBuild=false
+    # We don't commit the incremented version, but we use this to know the version when generating
+    # the resulting markdown
+    - name: Increment version
+      shell: bash
+      run: python build/versionutils.py gradle.properties --increment -u ${{ inputs.update_type }}
+    - name: Get new version
+      id: get_new_version
+      shell: bash
+      run: |
+        echo "version=$(python build/versionutils.py gradle.properties)" >> "$GITHUB_OUTPUT"
+    - name: Create results directory
+      shell: bash
+      run: mkdir artifact
+    - name: Create markdown
+      shell: bash
+      run: python build/publish-mixed-mode-results.py ${{ steps.get_new_version.outputs.version }} --run-link ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }} > artifact/mixed-mode-results.md
```

Comment:
Looking over `publish-mixed-mode-results.py`, I think this is probably fine, as the only thing it should print out are the mixed mode markdown information. I could also see the argument, though, that we should give it an explicit output argument (other than the "release notes" one that's already there) to make it less likely that someone accidentally prints some debugging information and we unwittingly put it into the final artifact.

Reply:
Yeah, that's probably a good callout.
I added a `--output` parameter and removed the code that allows it to inject into release notes.